integration-examples/apiScripts/README.md

```diff
@@ -6,3 +6,9 @@ This directory contains sample API Scripts that you can use to call the Splunk O
 
 # addEmailToDetectors.py
 This script will allow users to insert an email address to the notifications for one or more detectors. It works with the token.yaml file which contains optional and required values needed to run the script. Please refer to the comments in the token.yaml file for more details.
+
+# getMetricsForHost.py
+This script is used to find all the metrics for a given host.
```

Comment:
Would be really cool to mention any possible flags for a given script in this section also?

Reply:
:+1: